tests/integration/__init__.py

```diff
@@ -630,8 +631,16 @@ def use_cluster(cluster_name, nodes, ipformat=None, start=True, workloads=None,
                 else:
                     CCM_CLUSTER = CCMCluster(path, cluster_name, **ccm_options)
                     CCM_CLUSTER.set_configuration_options({'start_native_transport': True})
-                if Version(cassandra_version) >= Version('2.2'):
-                    CCM_CLUSTER.set_configuration_options({'enable_user_defined_functions': True})
+                if Version(cassandra_version) >= Version('4.1'):
+                    CCM_CLUSTER.set_configuration_options({
+                        'user_defined_functions_enabled': True,
+                        'scripted_user_defined_functions_enabled': True,
+                        'materialized_views_enabled': True,
+                        'sasi_indexes_enabled': True,
```

Comment:
Why do we need to enable sasi_indexes_enabled ?

Reply:
I don't have a clue, it's just same as the driver test was doing before for configuring cassandra, but in a new syntax.